riscv_isac/coverage.py

```diff
@@ -702,6 +701,9 @@ def compute_per_line(queue, event, cgf_queue, stats_queue, cgf, xlen, flen, addr
                     rs1_val = (rs1_hi_val << 32) | rs1_val
             elif rs1_type == 'x':
                 rs1_val = struct.unpack(sgn_sz, bytes.fromhex(arch_state.x_rf[nxf_rs1]))[0]
+                if inxFlg:
+                    rs1_val = struct.unpack(fsgn_sz, bytes.fromhex(arch_state.x_rf[nxf_rs1]))[0]
+                    define_sem(flen,iflen,rs1_val,"1",instr_vars)                    
```

Comment:
The `define_sem` need not be called for all the functions when `inxFlg` is called. The fcvt.*.[l|w][u] instructions have their source operands in integer format instead of floating point.